.travis.yml

```diff
@@ -0,0 +1,25 @@
+dist: trusty
+sudo: required
+language: python
+python:
+  - "3.5"
```

Comment:
no python 3.6 is that right?

Reply:
I just modified the build conf to use `matrix` and now it will build against both 3.5 and 3.6